### PR TITLE
Fix issues with Atlas leaking memory.

### PR DESCRIPTION
**Pull request by profan**
* Fix issue with Font Atlas leaking memory by adding a drop impl, reported in issue #627

Split this out from the other PR so at least this change would go in, since it seems mostly unproblematic :)

### CODE REVIEW

**Comment by profan** [comment]
I guess the only question here might be, maybe that delete function does not need to be public?

File: src/text/atlas.rs [hunk 1]
```diff
@@ -184,4 +190,8 @@ impl Atlas {
             );
         }
     }
+    fn delete(&mut self) {
+        let ctx = &mut get_context().quad_context;
+        ctx.delete_texture(self.texture);
```
Comment (not-fl3) [inline src/text/atlas.rs:195]: I believe we used to have `delete` on everything, but now, with Drop, should it really be a separate function, not just `drop`?

Comment (profan) [reply src/text/atlas.rs:195]: Yeah that's fair, it's probably an entirely unnecessary convention to follow now, I'll remove the delete function and just have the Drop impl :)

**Comment by not-fl3** [comment]
Thanks for PR! :+1:

**Pull request merged**